packages/components/data-table/src/data-table.spec.js

```diff
@@ -45,6 +45,16 @@ describe('DataTable', () => {
     expect(screen.queryByTestId('header-year')).toBeInTheDocument();
   });
 
+  it('should be able to find headers by the data-id', () => {
+    const rendered = render(<DataTable {...baseProps} />);
+    expect(
+      rendered.container.querySelector('[data-id="title"]')
+    ).toBeInTheDocument();
+    expect(
+      rendered.container.querySelector('[data-id="year"]')
+    ).toBeInTheDocument();
+  });
+
```

Comment:
I added these test just to make sure that `data-id` prop is working and is assigned to an element